README.md

```diff
@@ -1 +1,49 @@
-# sldt-semantic-models
+# Aspect Models for Eclipse Tractus-X Semantic Layer (SLDT)
+The repository contains the aspect models based on [BAMM](https://openmanufacturingplatform.github.io/sds-documentation/bamm-specification/snapshot/index.html) for the Tractus-X project in Catena-X.
+
+
+# Using the models
+The models can locally be processed with the [BAMM CLI](https://openmanufacturingplatform.github.io/sds-documentation/sds-documentation/index.html).
+It allows you to generate different artifacts (diagrams, example payload, java class files) out of it.
+
+To generate the commonly used set of artifacts, this repository also contains a script, `generate.sh`, which automates some steps, and might be useful for your task.
+
+Example call for the Material.ttl aspect
+```
+./generate.sh io.catenax.batch/1.0.2/Batch.ttl
+
+```
+You will find a folder Material-artifacts in the root folder of the repository, which contains the generated artifacts.
+
+Limitations of the script: You need an internet connection to allow the script to download the BAMM-CLI and the Catena-X style for the documentation. 
+Currently the script uses version 1.0.2 of the BAMM-CLI, which doesn't work properly on the Mac (graphics will not generate images [issue](https://github.com/OpenManufacturingPlatform/sds-sdk/issues/38)). 
```

Comment:
I think they fixed this issue and we can remove the remark

Reply:
Good point, I still kept a recommendations for which CLI version to use.